core/nuxeo-connection.js

```diff
@@ -163,6 +172,12 @@ import './nuxeo-element.js';
           };
         }
       } else {
+        if (this.token) {
```

Comment:
We should verify that the intended authentication method is indeed token:
```suggestion
        if (this.method === 'token' && this.token) {
```